bpf/lib/lb.h

```diff
@@ -283,15 +283,15 @@ static __always_inline bool lb_skip_l4_dnat(void)
 }
 
 static __always_inline
-bool lb4_svc_is_ext_local_scope(const struct lb4_service *svc)
+bool lb4_svc_is_two_scopes(const struct lb4_service *svc)
 {
-	return svc->flags & SVC_FLAG_EXT_LOCAL_SCOPE;
+	return (svc->flags & SVC_FLAG_EXT_LOCAL_SCOPE) != (svc->flags2 & SVC_FLAG_INT_LOCAL_SCOPE);
```

Comment:
I'm a bit puzzled by this check and the function name. Is this function supposed to check whether any scope is set (external, local, or both)? From the check it look that's the case. However, the name reads as if it checks for both being set. Maybe then `lb4_svc_has_any_scope()`?

Reply:
I think it's okay as is (maybe comment could help here). If both scopes are Local, then we don't use the two lookup scopes since the set of backends for both cases is the same.